src/test/regress/expected/create_null_dist_key.out

```diff
@@ -536,15 +543,19 @@ BEGIN;
   CREATE ROLE table_users;
   CREATE POLICY table_policy ON null_dist_key_table_3 TO table_users
       USING (table_user = current_user);
+  GRANT ALL ON TABLE null_dist_key_table_3 TO table_users;
+  ALTER TABLE null_dist_key_table_3 OWNER TO table_users;
```

Comment:
Do you plan to add post-create-distributed-table time tests for those in another PR?

Reply:
Added them into this one